components/skus/common/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+third_party/blink/public",
+  "+v8/include",
```

Comment:
same as above, v8 is not allowed in common

Reply:
Fixed